README.md

```diff
@@ -110,6 +122,22 @@ const checkoutInstance = startCheckout(manifest, {
 })
 ```
 
+### Changing appearance
+
+```js
+const checkoutInstance = easypayCheckout.startCheckout(manifest, {
+  logoUrl: 'www.example.com/mylogo.png',
+  accentColor: 'orange',
+  buttonBackgroundColor: '#111',
+  buttonBoxShadow: false,
+  buttonBorderRadius: 5,
+  inputBorderRadius: 5,
+  inputBorderColor: '#000',
+  inputBackgroundColor: '#ffe7c4',
+  backgroundColor: '#eee',
```

Comment:
I think we should add the fontFamily example too, just so the integrator doesn't misunderstand what should he pass.